src/App.tsx

```diff
@@ -15,48 +16,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
```

Comment:
```suggestion
enum SortType {
```

Reply:
<img width="748" alt="image" src="https://github.com/mate-academy/react_list-of-goods-hooks/assets/86789161/2526a523-1193-474f-8753-24143cee652d">